lldb/test/API/lang/swift/async/frame/variables_multiple_frames/TestSwiftAsyncFrameVarMultipleFrames.py

```diff
@@ -37,6 +37,27 @@ def check_pcs(self, async_frames, process, target):
             prologue_to_skip = parent_frame.GetFunction().GetPrologueByteSize()
             self.assertEqual(continuation_ptr + prologue_to_skip, parent_frame.GetPC())
 
+    def check_async_regs_one_frame(self, frame, process):
+        is_x86 = 'x86' in process.GetTarget().GetTriple()
```

Comment:
Jason suggested some variant of ` if self.getArchitecture() in ["x86_64"]:`

Reply:
could the architecture ever return `x86_64h`?

Reply:
I honestly did not know this was a thing!

Reply:
I've skipped the test on non `arm64, arm64e, x86_x64` archs (as done in other swift tests) and used Jason's suggestion